drivers/base/kdrosdbg/CMakeLists.txt

```diff
@@ -7,6 +7,15 @@ list(APPEND SOURCE
     ${CMAKE_CURRENT_BINARY_DIR}/kdcom.def)
 
 add_library(kdcom MODULE ${SOURCE})
+
+if(SANITIZE_UB OR KASAN_ENABLED)
```

Comment:
That whole thing certainly needs to be under a `if(CMAKE_C_COMPILER_ID STREQUAL "Clang")`, because MSVC (at least) doesn't support that.